alarm_trap_sender.cpp

```diff
@@ -50,35 +50,61 @@ AlarmFilter AlarmFilter::_instance;
 
 AlarmTrapSender AlarmTrapSender::_instance;
 
-bool ActiveAlarmList::update(AlarmTableDef& alarm_table_def, const std::string& issuer)
+bool ObservedAlarms::update(AlarmTableDef& alarm_table_def, const std::string& issuer)
 {
-  bool updated = false;
+  bool send_trap = false;
```

Comment:
I preferred the old name. The observed alarms class has no knowledge of traps - it can only say whether the observed alarms have changed, so updated was a good name for the variable.